.github/workflows/acceptance.yml

```diff
@@ -375,3 +375,57 @@ jobs:
             github-token: ${{ secrets.GITHUB_TOKEN }}
             gcs-bucket: 'sentry-visual-snapshots'
             gcp-service-account-key: ${{ secrets.SNAPSHOT_GOOGLE_SERVICE_ACCOUNT_KEY }}
+
+    openapi-diff:
```

Comment:
Let's move this to a new workflow file named `openapi-diff.yml` and job as `diff` or `check`